src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5291,6 +5291,7 @@ namespace HeatBalanceManager {
 
 				if ( MaxHeatLoadZone( ZoneNum ) > 1.0e-4 ) { // make sure load big enough to divide
 					MaxHeatLoadZone( ZoneNum ) = std::abs( max( MaxHeatLoadZone( ZoneNum ), MinLoad ) );
+					MaxHeatLoadPrevDay( ZoneNum ) = std::abs( max( MaxHeatLoadPrevDay( ZoneNum ), MinLoad ) );
```

Comment:
@EnergyArchmage that doesn't make sense to me at this point without knowing more, but I don't see that this hurts anything other than runtime. I know you are trying to improve execution speed so I'll take your word for it at this point.